### ISSUE TITLE
CI: Add CI-specific view

### ISSUE THREAD

**Issue by danielbeardsley** [iFixit/pulldasher#340]
#### Potential Solution
A view that shows a table of CI runs where the rows are pulls and the columns are CI statuses, click the column headers to sort the pulls  by when that CI status was finished (or started?)

Another option would be to only show one status at a time and have the rows be pulls and the columns be:

* Pull title (links to pull)
* Branch name
* CI time
* Status (Links to log)

This seems a bit more focused

**Comment by danielbeardsley** [comment]
:point_up: CC @iFixit/qae

**Comment by ardelato** [comment]
@danielbeardsley do you want to see this implemented particularly on `Pulldasher` or do you want to be able to find out that information in some way?

**Comment by mlahargou** [comment]
Is this what you're looking for?

https://metabase.ifixit.com/dashboard/47-ci-status

It's broken since we stopped writing to ifixit I believe.

**Comment by danielbeardsley** [comment]
> do you want to see this implemented particularly on `Pulldasher`

Yeah. We have all the data right there in the browser in the pull objects and it's live / kept up to date.

**Comment by danielbeardsley** [comment]
Yes, this, but allow choosing a particular CI status (one of the 19!)
![image](https://user-images.githubusercontent.com/26855/198116097-24c42750-755d-47eb-8020-e99cf619b4ba.png)

I'd imagine adding a link on each row here that would open the CI view with all the recent runs of that CI job:
![image](https://user-images.githubusercontent.com/26855/198116395-e8826e7a-fa2a-4566-b7a1-ae94daaa8477.png)

**Comment by danielbeardsley** [comment]
FYI, we have all the merged pulls from the last two weeks on the client side: https://github.com/iFixit/pulldasher/blob/master/app.js#L50-L58

So all the data is there.

**Comment by mlahargou** [comment]
Cool! Sounds good to me 👍

**Comment by mlahargou** [comment]
QAE doesn't plan on doing this any time soon. Pulling it off of our backlog.

**Comment by danielbeardsley** [comment]
I made a proof-of-concept in a branch:
https://github.com/iFixit/pulldasher/compare/ci-dashboard-proof-of-concept

But then I remembered that we only include CI runs / checks for the most recent commit of each pull. This definitely limits its usefulness because it doesn't represent the collection of recent runs but the collection of runs on recent commits.

It may still help answer the question: "Of the CI blocked pulls, which statuses are causing the blocks"
